receiver/prometheusreceiver/config_test.go

```diff
@@ -316,6 +316,17 @@ func TestFileSDConfigYamlNilTargetGroup(t *testing.T) {
 	require.NoError(t, component.ValidateConfig(cfg))
 }
 
+func TestTargetAllocatorInvalidHTTPScrape(t *testing.T) {
```

Comment:
Can you add a test (or extend an existing test) with a valid `http_scrape_config` section?

Reply:
Done.